django/contrib/admin/checks.py

```diff
@@ -916,9 +916,11 @@ def _check_list_display_item(self, obj, item, label):
                         id="admin.E108",
                     )
                 ]
-        if isinstance(field, models.ManyToManyField) or (
-            getattr(field, "rel", None) and field.rel.field.many_to_one
-        ):
+        if type(field) in [
+            models.ManyToManyField,
+            models.ManyToOneRel,
+            models.ManyToManyRel,
+        ] or (getattr(field, "rel", None) and field.rel.field.many_to_one):
```

Comment:
What do you think about using `is_relation`instead? It's more consistent with other checks
```suggestion
        if (
            getattr(field, "is_relation", False)
            and (field.many_to_many or field.one_to_many)
        ) or (getattr(field, "rel", None) and field.rel.field.many_to_one):
```

Reply:
Thanks for comments.
Done